python/cudf/cudf/utils/ioutils.py

```diff
@@ -1571,6 +1579,20 @@ def _get_filesystem_and_paths(path_or_data, storage_options):
     return fs, return_paths
 
 
+def _expand_directories(paths, glob_pattern, fs):
```

Comment:
I suppose the convention is to call this something like `_maybe_expand_directories`